README.md

```diff
@@ -1,36 +1,62 @@
 # Tic Tac Toe Ruby Game (The Odin Project)
 
-> This project is part of the Microverse Ruby Curriculum and The Oding Project. 
+> This is a Tic Tac Toe game made with Ruby. It's a part of the Microverse Ruby Curriculum and The Odin Project. 
 
-## Built With
+## Built using
 
 - Ruby
+- Gitflow Workflow
+
+## UML Class diagram
+
+![screenshot](./UML_Class_diagram.png)
+
+## Project requirements
+- All user interaction and game flow should be kept only in main.rb
+- Use OOP
 
 ## Live Demo
 [Live preview:](https://repl.it/join/gjrzjjbm-marcoshdezcam1)
```

Comment:
- [ ] **OPTIONAL** You can take out the colon (:) at the end of the line, it looks as if you are missing something